client/src/components/CompanyRegister.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from "react";
+import {
+  Form,
+  Input,
+  Dropdown,
+  Button,
+  TextArea,
+  Message,
+  Grid,
+  Header
+} from "semantic-ui-react";
+
+import { companyRegister } from "../api/companies";
+import { industryList } from "../utils/constants";
+
+class CompanyRegister extends Component {
+  state = {
+    companyProfile: {
+      name: "",
+      description: "",
+      industry: [],
+      email: "",
+      password: "",
+      user_id: ""
+    },
+    error: null,
+    success: null
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    companyRegister(this.state.companyProfile)
+      .then(data => {
+        console.log(data);
+        if (data.success) {
+          this.setState({
+            companyProfile: {
+              name: "",
+              description: "",
+              industry: [],
+              email: "",
+              password: ""
+            },
+            success: true
+          });
+        } else {
+          this.setState({});
+        }
+        this.resetForm(data.success);
+      })
+      .catch(err => {
+        this.setState({ success: false, error: err });
+      });
+  };
+
+  handleIndustry = (event, data) => {
+    const selectedIndustries = data.value;
+    this.setState({
+      companyProfile: {
+        ...this.state.companyProfile,
+        industry: selectedIndustries
+      }
+    });
+  };
+
+  resetForm = success => {
+    this.setState({
+      companyProfile: {
+        name: "",
+        description: "",
+        industry: [],
+        email: "",
+        password: ""
+      },
+      success: success
+    });
+  };
+  handleChange = event => {
+    const property = event.target.name;
```

Comment:
same as previous comment .. you can write with destructuring syntax like 
```
const { target } = event; // destructuring target from event
const { name: property } = target // destrucuring name and renaming it to property
```

More info about destructuring - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment